AirCasting/SDSync/FixedSessions/SDCardFixedSessionsSavingService.swift

```diff
@@ -75,19 +75,43 @@ class SDCardFixedSessionsUploadingService {
         }
     }
     
-    private func enqueueForUploading(measurements: SDCardMeasurementsRow, buffer streamsWithMeasurements: inout [SDStream: [Measurement]]) {
+    private func provideLines(of url: URL, progress: (FileLineReaderProgress) -> Void) throws {
+        var isDirectory: ObjCBool = false
+        guard FileManager.default.fileExists(atPath: url.path, isDirectory: &isDirectory) else { progress(.endOfFile); return }
+        
+        if !isDirectory.boolValue {
```

Comment:
too deep indentations here, let's make it a `guard isDirectory.boolValue` to get rid of one